README.md
# EasyUpdate3

Fork vom Contao Modul easyupdate.    
Ich habe das Modul für Contao 3 angepasst, um weiter damit arbeiten zu können.


## Installation über ER2

* [zum Contao ER2](https://contao.org/de/extension-list/view/easyupdate3.de.html)


## Installation über Composer

* Paketname suchen: `bugbuster/easyupdate3`


## For Translators
Translations are managed using Transifex. To create a new translation or to help to maintain an existing one, please register at transifex.com.

Project Link: [https://www.transifex.com/projects/p/contao-easyupdate3/](https://www.transifex.com/projects/p/contao-easyupdate3/)

Howto: [http://docs.transifex.com/faq/#translating](http://docs.transifex.com/faq/#translating)


## Icons
This software using the Fugue Icons from http://p.yusukekamiyamane.com/icons/search/fugue/


CHANGELOG.md
# Changelog EasyUpdate3

## [3.5.4](https://github.com/Contao3-Ninja/contao-easyupdate3/issues?q=milestone%3A3.5.4+is%3Aclosed) (2020-11-26)

### Fixed bugs

- [\#92](https://github.com/BugBuster1701/contao-easyupdate3/issues/92) Fixed "Server Offline" problem

## [3.5.3](https://github.com/BugBuster1701/contao-easyupdate3/issues?q=milestone%3A%223.5.3%22+is%3Aclosed) (2018-09-18)

### Fixed bugs

- [\#86](https://github.com/BugBuster1701/contao-easyupdate3/issues/86) Fixes some PHP 7.2+ problems

## [3.5.2](https://github.com/BugBuster1701/contao-easyupdate3/issues?q=milestone%3A%223.5.2%22+is%3Aclosed) (2018-01-24)

### Fixed bugs

- [\#81](https://github.com/BugBuster1701/contao-easyupdate3/issues/81) Undefined constant uncompressed_size
- [\#80](https://github.com/BugBuster1701/contao-easyupdate3/issues/80) License "LGPL-3.0+" is a deprecated SPDX license identifier, use "LGPL-3.0-or-later" instead

## [3.5.1](https://github.com/BugBuster1701/contao-easyupdate3/issues?q=milestone%3A%223.5.1%22+is%3Aclosed) (2018-01-23)

### Fixed bugs

- [\#78](https://github.com/BugBuster1701/contao-easyupdate3/issues/78) Using PHP 7.2.1: Warning: Use of undefined constant ENCODE_AMPERSANDS

## [3.5.0](https://github.com/BugBuster1701/contao-easyupdate3/issues?q=milestone%3A%223.5.0%22+is%3Aclosed) (2017-01-23)

### New features

- [\#71](https://github.com/BugBuster1701/contao-easyupdate3/issues/71) opcache Unterstützung
- [\#68](https://github.com/BugBuster1701/contao-easyupdate3/issues/68) Startpanel erweitern
- [\#49](https://github.com/BugBuster1701/contao-easyupdate3/issues/49) check .htaccess and .htaccess.default or give a hint to manually compare them
- [\#41](https://github.com/BugBuster1701/contao-easyupdate3/issues/41) PHP Version prüfen für Zielversion

### Fixed bugs

- [\#73](https://github.com/BugBuster1701/contao-easyupdate3/issues/73) Anpassungen an Backend-Theme "flexible_c4" für Contao 3.5

## [3.4.0](https://github.com/BugBuster1701/contao-easyupdate3/issues?q=milestone%3A%223.4.0%22+is%3Aclosed) (2016-12-27)

### New features

- [\#33](https://github.com/BugBuster1701/contao-easyupdate3/issues/33) Composer Anpassungen

## [3.3.3](https://github.com/BugBuster1701/contao-easyupdate3/issues?q=milestone%3A%223.3.3%22+is%3Aclosed) (2016-04-30)

### Fixed bugs

- [\#63](https://github.com/BugBuster1701/contao-easyupdate3/issues/63) Cannot set max_execution_time above master value of 45 (tried to set 120)

## [3.3.2](https://github.com/BugBuster1701/contao-easyupdate3/issues?q=milestone%3A%223.3.2%22+is%3Aclosed) (2015-11-11)

### Fixed bugs

- [\#55](https://github.com/BugBuster1701/contao-easyupdate3/issues/55) Update Transifex Language Files
- [\#54](https://github.com/BugBuster1701/contao-easyupdate3/issues/54) Cache löschen an spezieller Stelle

## [3.3.1](https://github.com/BugBuster1701/contao-easyupdate3/issues?q=milestone%3A%223.3.1%22+is%3Aclosed) (2015-11-01)

### Fixed bugs

- [\#53](https://github.com/BugBuster1701/contao-easyupdate3/issues/53) "n.a." durch Text ersetzen, Update Sprachfiles
- [\#52](https://github.com/BugBuster1701/contao-easyupdate3/issues/52) Aufruf von purgeScriptCache
- [\#51](https://github.com/BugBuster1701/contao-easyupdate3/issues/51) Leere Auswahl führt zur Fehlermeldung
- [\#50](https://github.com/BugBuster1701/contao-easyupdate3/issues/50) clear div sometimes not present

## [3.3.0](https://github.com/BugBuster1701/contao-easyupdate3/issues?q=milestone%3A%223.3.0%22+is%3Aclosed) (2015-10-09)

### New features

- [\#44](https://github.com/BugBuster1701/contao-easyupdate3/issues/44) ReConnect anstoßen bei Connection Error
- [\#42](https://github.com/BugBuster1701/contao-easyupdate3/issues/42) Transfer  bei SMH nicht möglich
- [\#38](https://github.com/BugBuster1701/contao-easyupdate3/issues/38) Feature: Offline-Grund des Servers anzeigen
- [\#34](https://github.com/BugBuster1701/contao-easyupdate3/issues/34) Check auf Vorhandensein von Tivoka

### Fixed bugs

- [\#56](https://github.com/BugBuster1701/contao-easyupdate3/issues/56) Dbafs::addResource
- [\#46](https://github.com/BugBuster1701/contao-easyupdate3/issues/46) Transfer ZIP in DBFS eintragen
- [\#45](https://github.com/BugBuster1701/contao-easyupdate3/issues/45) File not found legt zip an

## [3.2.0](https://github.com/BugBuster1701/contao-easyupdate3/issues?q=milestone%3A%223.2.0%22+is%3Aclosed) (2015-01-24)

### New features

- [\#26](https://github.com/BugBuster1701/contao-easyupdate3/issues/26) Links zu update Files

## [3.1.0](https://github.com/BugBuster1701/contao-easyupdate3/issues?q=milestone%3A%223.1.0%22+is%3Aclosed) (2014-12-07)

### New features

- [\#31](https://github.com/BugBuster1701/contao-easyupdate3/issues/31) Textanpassungen
- [\#30](https://github.com/BugBuster1701/contao-easyupdate3/issues/30) Sortierung der gefundenen Update Files
- [\#27](https://github.com/BugBuster1701/contao-easyupdate3/issues/27) Liste der zu schützenden Dateien verringern
- [\#21](https://github.com/BugBuster1701/contao-easyupdate3/issues/21) Automatisch alte Dateien löschen.

### Fixed bugs

- [\#11](https://github.com/BugBuster1701/contao-easyupdate3/issues/11) Could not decompress data on 0 byte files



<sub>*This changelog was automatically generated by [bb_changelog_gen](https://github.com/BugBuster1701/bb_changelog_gen)*</sub>


languages/ru/default.php
<?php
/**
 * Translations are managed using Transifex. To create a new translation
 * or to help to maintain an existing one, please register at transifex.com.
 *
 * @link http://help.transifex.com/intro/translating.html
 * @link https://www.transifex.com/projects/p/contao-easyupdate3/language/ru/
 *
 * @license http://www.gnu.org/licenses/lgpl-3.0.html LGPL
 *
 * last-updated: 2015-07-30T00:29:05+02:00
 */

$GLOBALS['TL_LANG']['MOD']['easyupdate3']['0']                      = 'easyUpdate3';
$GLOBALS['TL_LANG']['MOD']['easyupdate3']['1']                      = 'Обновление Contao 3 из панели управления.';
$GLOBALS['TL_LANG']['easyupdate3']['all']                           = 'Выбрать все';
$GLOBALS['TL_LANG']['easyupdate3']['allow_url_fopen_not_set']       = 'Флаг PHP "allow_url_fopen" не установлен.';
$GLOBALS['TL_LANG']['easyupdate3']['backBT']                        = 'Назад';
$GLOBALS['TL_LANG']['easyupdate3']['backup']                        = 'Резервное копирование файлов';
$GLOBALS['TL_LANG']['easyupdate3']['backuped']                      = 'Резервные копии:';
$GLOBALS['TL_LANG']['easyupdate3']['changelog']['headline']         = 'Список изменений между версиями %s и %s';
$GLOBALS['TL_LANG']['easyupdate3']['changelog']['no']               = 'Невозможно прочитать список изменений. За дополнительной информацией вы можете обратиться к Contao-Community (http://www.contao.org).';
$GLOBALS['TL_LANG']['easyupdate3']['changelog']['same']             = 'Будет установлена та же самая версия, так что ничего не изменится.';
$GLOBALS['TL_LANG']['easyupdate3']['config_legend']                 = 'Файлы конфигурации';
$GLOBALS['TL_LANG']['easyupdate3']['content']                       = 'Содержимое архива';
$GLOBALS['TL_LANG']['easyupdate3']['curl_not_available']            = 'Расширение PHP CURL не доступно.';
$GLOBALS['TL_LANG']['easyupdate3']['delete']                        = 'Удаление старых файлов';
$GLOBALS['TL_LANG']['easyupdate3']['deleted']                       = 'Удаленные:';
$GLOBALS['TL_LANG']['easyupdate3']['demo']                          = 'Демо-файлы';
$GLOBALS['TL_LANG']['easyupdate3']['description']                   = 'Выберите архив из каталога (files/easyupdate3). Скрипт работает только с ZIP-архивами и не видит вложенные папки.';
$GLOBALS['TL_LANG']['easyupdate3']['done']                          = 'Готово!';
$GLOBALS['TL_LANG']['easyupdate3']['error']                         = 'Ошибки:';
$GLOBALS['TL_LANG']['easyupdate3']['exclude']                       = 'Исключены';
$GLOBALS['TL_LANG']['easyupdate3']['extern_notice']                 = 'Официальных обновлений нет. Нет поддержки от разработчиков Contao!';
$GLOBALS['TL_LANG']['easyupdate3']['extern_title']                  = 'Внешние, доступные архивы обновлений';
$GLOBALS['TL_LANG']['easyupdate3']['files']['backup']               = 'Файлы резервных копий';
$GLOBALS['TL_LANG']['easyupdate3']['files']['original']             = 'Оригинальные файлы';
$GLOBALS['TL_LANG']['easyupdate3']['files_not_availabe']            = 'файлы не доступны';
$GLOBALS['TL_LANG']['easyupdate3']['fopen_curl_notice']             = 'Расширению "easyUpdate3" необходим флаг "allow_url_fopen", который необходимо установить в файле php.ini или расширение PHP Curl, для связи с сервером easyUpdate3.';
$GLOBALS['TL_LANG']['easyupdate3']['get_next_update_error']         = 'Проблема поиска следующего обновления.';
$GLOBALS['TL_LANG']['easyupdate3']['get_next_update_file_notfound'] = 'Соответствующий файл не найден по UUID. Поставщик файлов обновлений был автоматически уведомлен об этой ошибке.';
$GLOBALS['TL_LANG']['easyupdate3']['get_next_update_found']         = 'Найдено подходящее обновление до версии %s';
$GLOBALS['TL_LANG']['easyupdate3']['get_next_update_notfound']      = 'Подходящих обновлений не найдено.';
$GLOBALS['TL_LANG']['easyupdate3']['get_next_update_url_error']     = 'Проблема поиска имени файла.';
$GLOBALS['TL_LANG']['easyupdate3']['headline']                      = 'easyUpdate обновление Contao (текущая версия: %s)';
$GLOBALS['TL_LANG']['easyupdate3']['log_notice']                    = 'Файл журнала можно найти с помощью файл-менеджера в каталоге: %s';
$GLOBALS['TL_LANG']['easyupdate3']['next']                          = 'Следующий шаг';
$GLOBALS['TL_LANG']['easyupdate3']['nothing_to_delete']             = 'Нет файлов для удаления.';
$GLOBALS['TL_LANG']['easyupdate3']['noupdate']                      = 'ИСКЛЮЧИТЬ файлы из обновления';
$GLOBALS['TL_LANG']['easyupdate3']['noupdatetext']                  = 'Вы можете выбрать файлы, ИСКЛЮЧАЕМЫЕ из обновления.<br>В резервную копию войдут все файлы включая эти.';
$GLOBALS['TL_LANG']['easyupdate3']['other_legend']                  = 'Другие файлы';
$GLOBALS['TL_LANG']['easyupdate3']['previous']                      = 'Вернуться назад';
$GLOBALS['TL_LANG']['easyupdate3']['readme']['headline']            = 'Прочтите полностью перед использованием';
$GLOBALS['TL_LANG']['easyupdate3']['readme']['text1_text']          = 'Получение списка файлов в архиве. Резервное копирование имеющихся файлов. Обновление файлов. Удаление старых файлов. Запуск мастера установки. Вполне возможно, что вам необходимо будет обновить базу данных Contao, в этом случае, вы увидите мастер установки.';
$GLOBALS['TL_LANG']['easyupdate3']['readme']['text1_title']         = 'Обновление состоит из 5 шагов:';
$GLOBALS['TL_LANG']['easyupdate3']['readme']['text2_text']          = 'К сожалению, ничто не является безопасным на 100%. Будьте осторожны с этим инструментом. Обязательно создайте резервную копию базы данных, прежде чем запускать этот инструмент в первый раз. Расширение производит изменения файловой системы, при этом возможна потеря данных. Все, что вы делаете, на ваш страх и риск.';
$GLOBALS['TL_LANG']['easyupdate3']['readme']['text2_title']         = 'Уведомление безопасности:';
$GLOBALS['TL_LANG']['easyupdate3']['readme']['text3']['incorrect']  = 'Проверено и не корректно работает: 2.x.x ==> 2.x.x';
$GLOBALS['TL_LANG']['easyupdate3']['readme']['text3']['working']    = 'Проверено и работает: 3.x.x ==> 3.x.x';
$GLOBALS['TL_LANG']['easyupdate3']['readme']['text4']               = 'Работа расширения была проверена на Contao версии 3.4.0. Работа с Contao 2.x невозможна!. Если вы видите версию XXX, вполне возможно, что ZIP-архив поврежден.';
$GLOBALS['TL_LANG']['easyupdate3']['selectfile']                    = 'Выберите локальный архив (ZIP-файл)';
$GLOBALS['TL_LANG']['easyupdate3']['server_error']                  = 'Сервер не доступен';
$GLOBALS['TL_LANG']['easyupdate3']['server_error_notice']           = 'Причина заносится в "Системный журнал" раздела "Система".';
$GLOBALS['TL_LANG']['easyupdate3']['server_offline']                = 'Сервер не доступен';
$GLOBALS['TL_LANG']['easyupdate3']['server_online']                 = 'Сервер доступен';
$GLOBALS['TL_LANG']['easyupdate3']['setfile']                       = 'Выбрать';
$GLOBALS['TL_LANG']['easyupdate3']['skipped']                       = 'Пропущены:';
$GLOBALS['TL_LANG']['easyupdate3']['smh_warning']                   = 'Внимание: "Safe Mode Hack" обнаружен, но не поддерживается.';
$GLOBALS['TL_LANG']['easyupdate3']['update']                        = 'Обновление ваших файлов';
$GLOBALS['TL_LANG']['easyupdate3']['update0']                       = 'Будет установлена старая версия Contao. Вы уверены в необходимости этого? Обновление на более раннюю версию, может быть не легким занятием.';
$GLOBALS['TL_LANG']['easyupdate3']['update1']                       = 'Вы установите более новую версию Contao.';
$GLOBALS['TL_LANG']['easyupdate3']['update2']                       = 'Вы устанавливаете ту же самую версию.';
$GLOBALS['TL_LANG']['easyupdate3']['update_download']               = 'Загрузить';
$GLOBALS['TL_LANG']['easyupdate3']['update_transfer']               = 'Перенести';
$GLOBALS['TL_LANG']['easyupdate3']['update_transfer_result_notok']  = 'Проблема при переносе файла %s';
$GLOBALS['TL_LANG']['easyupdate3']['update_transfer_result_ok']     = 'Файл %s был перенесен и теперь может быть выбран слева.';
$GLOBALS['TL_LANG']['easyupdate3']['update_transfer_to']            = 'Перенести в %s';
$GLOBALS['TL_LANG']['easyupdate3']['update_transfer_wrong_hash']    = 'Проблема при переносе файла %s, неправильный MD5-хеш.';
$GLOBALS['TL_LANG']['easyupdate3']['updated']                       = 'Обновлены:';
$GLOBALS['TL_LANG']['easyupdate3']['updatex']                       = 'версия (%s) ======> версия (%s)';

